doc/plutus/tutorials/EscrowImpl.hs

```diff
@@ -308,7 +308,7 @@ redeem inst escrow = mapError (review _EscrowError) $ do
                 <> Constraints.mustValidateIn valRange
     if current >= escrowDeadline escrow
     then throwing _RedeemFailed DeadlinePassed
-    else if foldMap (view Tx.ciTxOutValue) unspentOutputs `lt` targetTotal escrow
```

Comment:
Why are we dropping lenses?

Reply:
It wasn't intentional, I'll bring this up in chat so we can discuss.